docs/examples/bifacial/plot_bifi_model_mc.py

```diff
@@ -0,0 +1,96 @@
+"""
+Bifacial Modeling - modelchain
+==============================
+
+Example of bifacial modeling using the modelchain
+"""
+
+# %%
+# This example shows how to complete a bifacial modeling example using the
+# :py:class:`pvlib.modelchain.ModelChain` with the
+# :py:func:`pvlib.bifacial.pvfactors_timeseries` function to transpose
+# GHI data to both front and rear Plane of Array (POA) irradiance.
+
+import pandas as pd
+from pvlib import pvsystem
+from pvlib import location
+from pvlib import modelchain
+from pvlib.temperature import TEMPERATURE_MODEL_PARAMETERS as PARAMS
+from pvlib import bifacial
+
+# create site location and times characteristics
+lat, lon = 36.084, -79.817
+tz = 'Etc/GMT+5'
+times = pd.date_range('2021-06-21', '2021-6-22', freq='1T', tz=tz)
+
+# create site system characteristics
+axis_tilt = 0
+axis_azimuth = 180
+gcr = 0.35
+max_angle = 60
+pvrow_height = 3
+pvrow_width = 4
+albedo = 0.2
+
+# load temperature parameters and module/inverter specifications
+temp_model_parameters = PARAMS['sapm']['open_rack_glass_polymer']
```

Comment:
```suggestion
temp_model_parameters = PARAMS['sapm']['open_rack_glass_glass']
```

The glass-glass coeffs are probably more appropriate for bifacial mods, though I'll defer to the Sandia folks on that